src/components/inline_edit/inline_edit_form.test.tsx

```diff
@@ -88,6 +88,32 @@ describe('EuiInlineEditForm', () => {
       expect(container.firstChild).toMatchSnapshot();
       expect(getByText('This is a placeholder.')).toBeTruthy();
     });
+
+    test('title for defaultValue', () => {
+      const { container, getByTestSubject } = render(
+        <EuiInlineEditForm {...commonInlineEditFormProps} />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+      expect(
+        getByTestSubject('euiInlineReadModeButton').getAttribute('title')
+      ).toEqual('Hello World!');
```

Comment:
My 2c - let's skip the snapshot on this and go for a more specific assertion. We should also be taking advantage of RTL's [getByTitle query API](https://testing-library.com/docs/queries/bytitle/) here:

```suggestion
      const { getByTitle } = render(
        <EuiInlineEditForm {...commonInlineEditFormProps} />
      );

      expect(getByTitle('Hello World!')).toBeTruthy();
```